app/src/main/java/com/a1ishka/bankaccount/navigation/NavGraph.kt

```diff
@@ -0,0 +1,46 @@
+package com.a1ishka.bankaccount.navigation
+
+import androidx.compose.runtime.Composable
+import androidx.navigation.NavType
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+import androidx.navigation.navArgument
+import com.a1ishka.bankaccount.screens.accountDashboard.AccountDashboard
+import com.a1ishka.bankaccount.screens.allTransactions.AllTransaction
+import com.a1ishka.bankaccount.screens.newAccountScreen.NewAccountScreen
+import com.a1ishka.bankaccount.screens.newTransaction.NewTransactionScreen
+
+@Composable
+fun NavGraph() {
+    val navController = rememberNavController()
+    NavHost(navController = navController, startDestination = Screen.AccountDashboardScreen.route) {
+        composable(route = Screen.AccountDashboardScreen.route) {
+            AccountDashboard(navController = navController)
+        }
+
+        composable(route = Screen.AccountAddingScreen.route) {
+            NewAccountScreen()
+        }
+
+        composable(route = Screen.AllTransactionScreen.route) {
+            AllTransaction()
+        }
+
+        composable(
+            route = Screen.TransactionDetailsScreen.route + "?id=${id}",
+            arguments = listOf(
+                navArgument("id") {
```

Comment:
set names of arguments to constants to avoid mistakes

Reply:
added 

`package com.a1ishka.bankaccount.util

object Constants {
    const val ARGUMENT_ID = "id"
}`